src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -236,10 +235,64 @@ public Task Performance()
             return true;
         }
 
+        [RavenAction("/databases/*/replication/pulses/live", "GET", AuthorizationStatus.ValidUser, SkipUsagesCount = true)]
+        public async Task PulsesLive()
```

Comment:
`event` and `taskEvent` are so generic we might as well call them `item1` 
Let us cal this `replication/live-tracking` or something like this?